user_interaction/templates/user_interaction/home_users.html

```diff
@@ -0,0 +1,118 @@
+                                                             margin-left: -1em; margin-top: -0.5em;"
+                                                    ></div>
+                                                </div>
+                                            {% endif %}
+                                        </div>
+                                        <div>
+                                            {{ group.name }}
+                                        </div>
+                                    </div>
+                                    <ul class="list-group list-group-flush">
+                                        <li class="list-group-item">
+                                            <div class="row">
+                                                <div class="col">
+                                                    Control board
+                                                </div>
+                                                <a class="col btn btn-sm btn-outline-primary"
+                                                   href="{% url "committees:group_general" group_id=group.id %}"
+                                                >   Go to
+                                                </a>
+                                            </div>
+                                        </li>
+                                        {% for shortcut in group.shortcut_set.all%}
+                                            <li class="list-group-item">
+                                                <div class="row">
+                                                    <div class="col">
+                                                        {{ shortcut.name|capfirst }}
+                                                    </div>
+                                                    <a class="col btn btn-sm btn-outline-primary"
+                                                       href="{{ shortcut.url }}"
+                                                    >   Go to
+                                                    </a>
+                                                </div>
+                                            </li>
+                                        {% endfor %}
+                                    </ul>
+                                </div>
+                            {% endfor %}
+                        </div>
+                    {% endif %}
+                {% endif %}
+
+                <div>
+                    <h4>Encountered evil?</h4>
+                    <p>
+                        Bugs are annoying. If you encountered one, please mark them on github or notify the board and
+                        we will send in our troops!
+                    </p>
+                    <a class="btn btn-outline-dark" href="https://github.com/esrg-knights/Squire/issues">Report bug</a>
+                </div>
+            </div>
+        </div>
+    </div>
+{% endblock %}
```

Comment:
Privacy policy notice is missing

Reply:
Notification is only visible when not logged in. Don't really see a use of placing it as a reminder constantly when you already accepted them. That said, this made me realise of a bigger problem that all those documents are currently hidden away too much. So I made #155. Also, did you know that on the registration page we do not mention the privacy document?

Reply:
> Also, did you know that on the registration page we do not mention the privacy document?
Good point! The mention should be there somewhere.